Dockerfile-cli

```diff
@@ -14,7 +16,9 @@ COPY src/php/utils/docker/ /usr/local/bin/
 
 # Install PHP extensions
 # hadolint ignore=DL4006
-RUN set -x \
+RUN apk update \
+    && apk upgrade \
+    && set -x \
```

Comment:
This will bloat the Docker image by installing the apk repo metadata. Instead, do not run `apk update` (unnecessary anyway) and use `apk upgrade --no-cache`, similar to what we do with `apk add` below.

Also, `set -x` needs to come before `apk upgrade --no-cache`, otherwise failures will go unnoticed.